.github/workflows/conda_unit_test.yml

```
@@ -71,6 +71,6 @@ jobs:
     - name: Test with coverage and pytest
       shell: bash -l {0}
       run: |
-        coverage run --concurrency=thread --parallel-mode -m pytest -vvv
+        coverage run --concurrency=thread --parallel-mode -m pytest -vvv  # --exitfirst .
```

I believe this is what @mrakitin suggested to exit with failure code upon first failure

```suggestion
        set -vxeuo pipefail
        coverage run --concurrency=thread --parallel-mode -m pytest -vvv
```